pennylane/qnodes/base.py

```diff
@@ -234,6 +235,51 @@ def print_applied(self):
             else:
                 print("{}({}(wires={}))".format(return_type, op.name, op.wires))
 
+    def draw(self, charset="unicode", show_variable_names=False):
+        """Draw the QNode as a circuit diagram.
+
+        Consider the following circuit as an example:
+
+        .. code-block:: python
```

Comment:
```suggestion
        .. code-block:: python3
```

Reply:
This distinction is needed here, as python (a shortcut to the python2 syntax highlighter) didn't have the `@` operator, so it will fail at the tensor observable